contrib/test/integration/build/runc.yml

```diff
@@ -4,6 +4,7 @@
   git:
     repo: "https://github.com/opencontainers/runc.git"
     dest: "{{ ansible_env.GOPATH }}/src/github.com/opencontainers/runc"
+    version: "84a082bfef6f932de921437815355186db37aeb1"
```

Comment:
Let us try to use a version just before the seccomp commit in runc that broke us. That way we are close to master and can switch back once it is fixed.

Reply:
Lets do that in a separate PR. Allow this one in. so we can get in the 1.8 patch since the AMIs are just about to work.  @mrunalp Does the test ami satisfy the e2e and integration tests.  @cevich says he believes all of these tests are run when you test ami?

Reply:
@rhatdan Yes, the ami jobs will run everything. Once they pass we have to tag new amis which I can do.

Reply:
Great.